app/assets/stylesheets/components/forms.scss

```diff
@@ -57,12 +57,13 @@ textarea {
     box-shadow: none;
     -webkit-box-shadow: none;
     background: transparent;
+    line-height: 1.5;
   }
 }
 .icon-input {
   position: relative;
   input {
-    padding-left: 3rem;
+    padding-left: 3rem !important;
```

Comment:
just a question: was there a reasonable way to do this without having to resort to using `!important`?